docs/self-managed/setup/deploy/amazon/amazon-eks/eksctl.md

```diff
@@ -291,152 +334,98 @@ For detailed examples, review the [documentation provided by AWS](https://docs.a
-    --db-cluster-identifier $RDS_NAME \
-    --engine aurora-postgresql \
-    --engine-version $POSTGRESQL_VERSION \
-    --no-publicly-accessible \
-    --db-instance-class db.t3.medium
+cat << EOF | kubectl apply -f -
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: ebs-sc
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+provisioner: ebs.csi.aws.com
+parameters:
+  type: gp3
+reclaimPolicy: Retain
+volumeBindingMode: WaitForFirstConsumer
+EOF
 ```
 
-More configuration options can be found in the [AWS documentation](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/rds/create-db-instance.html).
-
-12. Wait for changes to be applied:
+2. **Modify the `gp2` StorageClass to mark it as a non-default StorageClass:**
 
 ```shell
-aws rds wait db-instance-available \
-    --db-instance-identifier $RDS_NAME
+kubectl patch storageclass gp2 -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
 ```
 
-### Verifying connectivity between the Amazon EKS cluster and the PostgreSQL database
+After executing these commands, you will have a `gp3` StorageClass set as the default and the `gp2` StorageClass marked as non-default, provided that **gp2** was already present. Ensure to verify the changes again by running the `kubectl get storageclass` command.
 
-1. Retrieve the writer endpoint of the DB cluster.
+### Requirements for a domain deployment
 
-```shell
-export DB_HOST=$(aws rds describe-db-cluster-endpoints \
-  --db-cluster-identifier $RDS_NAME \
-  --query "DBClusterEndpoints[?EndpointType=='WRITER'].Endpoint" \
-  --output text)
-```
+If you plan to deploy Camunda using an external domain associated with an external certificate, you will need to set up some IAM policies to allow both **external-dns** and **cert-manager** to interact with Route 53, which controls the DNS.
 
-2. Start Ubuntu container in interactive mode within the Amazon EKS cluster.
+By default, the cluster uses **Pod Identity** to manage IAM roles for your applications. This means that service accounts are associated with IAM roles, allowing your pods to securely access AWS resources without hardcoding credentials. For more information on configuring Pod Identity, refer to the [AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/pod-identities.html).
 
-```shell
-kubectl run ubuntu --rm -i --tty --image ubuntu --env="DB_HOST=$DB_HOST" --env="PG_USERNAME=$PG_USERNAME" -- bash
-```
+Additionally, to [enable OpenID Connect (OIDC) and IAM Roles for Service Accounts (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) on your cluster, you can use the following commands:
 
-3. Install required dependencies:
+1. **OIDC Issuer:**
 
-```shell
-apt update && apt install -y postgresql-client
-```
+   First, ensure that your EKS cluster is set up with an OIDC provider. The following command should show you the OIDC issuer:
```

Comment:
the phrasing is a bit off.

> First, ensure that your EKS cluster is set up with an OIDC provider.

followed by step 2

> Create an OIDC provider

--- 

From https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html

you first retrieve the OIDC issuer ID, optionally check whether it exists, then create provider.